grpc-proto/src/main/java/io/stargate/grpc/CqlHelper.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.grpc;
+
+import stargate.Cql.Relation;
+import stargate.Cql.Relation.Target;
+import stargate.Cql.Relation.Target.TokenTarget;
+import stargate.Cql.Select;
+import stargate.Cql.Selector;
+import stargate.Cql.Selector.FunctionCallSelector;
+import stargate.Cql.Statement;
+import stargate.Cql.Term;
+
+/** Helper methods to build common patterns in CQL queries. */
+public class CqlHelper {
+
+  public static Statement newStatement(Select.Builder selectBuilder) {
+    return Statement.newBuilder().setSelect(selectBuilder).build();
+  }
+
+  public static Selector.Builder selectColumn(String columnName) {
+    return Selector.newBuilder().setColumnName(columnName);
+  }
```

Comment:
The goal here is to simplify the 80% most common use cases.

Selecting a column is super common.
Calling a function on a map element where the map is a UDT field is technically possible, but not likely so if the client needs it they can write the full boilerplate.

Reply:
How complicated you think would be that we can have a DSL like helper for each statement type? This would effectively be even better than a builder? Meaning you can program like:

```
select()
   .keyspace(..)
   .table(..)
   .column(..)
   .and()
   .column(..)
```

I think this is a huge developer benefit.. Instead of mixing the builder and helper, create our own DSL on top of the builder..

Reply:
They are a great example of easy to use DSL -> https://github.com/awaitility/awaitility

Reply:
I don't know, that sounds like a lot of work, essentially double the effort to add another layer. I'd rather keep things simple.